diff --git a/node_modules/lerna/dist/index.js b/node_modules/lerna/dist/index.js
index 4c494bc..bcabc20 100644
--- a/node_modules/lerna/dist/index.js
+++ b/node_modules/lerna/dist/index.js
@@ -3164,7 +3164,7 @@ async function createProjectGraphWithPackages(projectGraph, projectFileMap, pack
   const _workspaceRoot = process.env["NX_WORKSPACE_ROOT_PATH"] || import_devkit3.workspaceRoot;
   const projectNodes = Object.values(projectGraph.nodes);
   const projectNodesMatchingPackageConfigs = projectNodes.filter((node) => {
-    const matchesRootPath = (config) => (0, import_minimatch2.default)(node.data.root, config);
+    const matchesRootPath = (config) => (0, import_minimatch2.minimatch)(node.data.root, config);
     return packageConfigs.some(matchesRootPath);
   });
   const tuples = await Promise.all(
